test/utilities.py

```diff
@@ -35,55 +31,31 @@ def assert_pdf_equal(
         test (unittest.TestCase)
         pdf_or_tmpl: instance of `FPDF` or `Template`. The `output` or `render` method will be called on it.
         rel_expected_pdf_filepath (str): relative file path to a PDF file matching the expected output
-        delete (bool): clean up temporary PDF files after performing test
```

Comment:
This was not mentioned in the docstring description, but the purpose of this parameter was to allow to easily study the `qpdf-qdf`-formatted PDF files upon failure.

With the new file-less system, it's note possible to replicate this,
so I guess I'll add some lines instead in `docs/Development.md` to explain how to manually do this using the `qpdf` CLI

Reply:
In 99.9% of cases the test is green, so it's good to have it fast.

I could export the data to `/tmp/pytest-of-$USER` where it can be found easily for further investigation.

Each time? Or maybe there's a way to do some operation in the case of an error/failure?

Reply:
Sounds like a good idea!
Actually you could preserve the `delete` parameter and do exactly that if it is set to `True`.
It would be also useful to display the file paths in that case, as it used to be.

Reply:
+1 from me too, doesnt have to be this PR - worst comes to worst people add hacks to save the files somewhere to inspect them for tests theyre working on

Reply:
Please see #84.